src/transaction/boot_sr.c

```diff
@@ -4253,7 +4253,7 @@ xboot_copy (REFPTR (THREAD_ENTRY, thread_p), const char *from_dbname, const char
 	  else
 	    {
 	      er_set (ER_WARNING_SEVERITY, ARG_FILE_LINE, ER_BO_DIRECTORY_DOESNOT_EXIST, 1, p);
-	      if (mkdir (p, 0777) < 0)
+	      if (mkdir (p, 0700) < 0)
```

Comment:
I don't really now this issue, but `xboot_backup()` also seems to create the lob directory:

```
4249       if (es_type == ES_POSIX && p != NULL)
4250         {
4251 #if defined (WINDOWS)
4252           if (realpath (p, fixed_pathbuf) != NULL
4253               && (stat (fixed_pathbuf, &stat_buf) == 0 && S_ISDIR (stat_buf.st_mode)))
4254 #else
4255           if (realpath (p, fixed_pathbuf) != NULL)
4256 #endif
4257             {
4258               strcpy (p, fixed_pathbuf);
4259             }
4260           else
4261             {
4262               er_set (ER_WARNING_SEVERITY, ARG_FILE_LINE, ER_BO_DIRECTORY_DOESNOT_EXIST, 1, p);
4263               if (mkdir (p, 0777) < 0)
4264                 {
4265                   cub_dirname_r (p, fixed_pathbuf, PATH_MAX);
4266                   er_set_with_oserror (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_ES_GENERAL, 2, "POSIX", fixed_pathbuf);
4267                   error_code = ER_BO_DIRECTORY_DOESNOT_EXIST;
4268                   goto error;
4269                 }
4270               else
4271                 {
4272                   er_clear ();
4273                 }
4274             }
4275         }
```

Don't we need to cover it?

Reply:
I do not find any link between `xboot_backup` and creating the lob directory? Can you please elaborate?
I have already reported the issue with `xboot_copy` upstream - see comments in http://jira.cubrid.org/browse/CBRD-24103 .

Reply:
?? me neither. I can't find. What happened to me? Sorry to bother you 😂